Dockerfile.arm64

```diff
@@ -0,0 +1,71 @@
+# ------------------------------------------------------------------------------
+# Build avalanche
+# ------------------------------------------------------------------------------
+FROM arm64v8/golang:1.20.8-bullseye AS avalanche
```

Comment:
The minimum supported version in AvalancheGo is 1.20.10: https://github.com/ava-labs/avalanchego/blob/2d8222da82114088a2a99d8d69f40846f1ea6208/scripts/build_avalanche.sh#L30